ParkingSpaceReservation/src/main/resources/application.properties

```diff
@@ -2,4 +2,5 @@ spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:mysql://${MYSQL_HOST:localhost}:3306/parking_space_reservation
 spring.datasource.username=user123
 spring.datasource.password=ps123
-spring.jpa.generate-ddl=true
\ No newline at end of file
+spring.jpa.generate-ddl=true
+spring.jpa.show-sql=true
```

Comment:
do we need this still ? maybe we can do hibernate logging if or when necessary instead of show sql 
[Read here](https://stackoverflow.com/questions/30118683/how-to-log-sql-statements-in-spring-boot)